src/main/resources/data/miningmaster/advancements/miningmaster/geo_dude.json

```diff
@@ -91,13 +91,37 @@
         ]
       }
     },
-    "miningmaster:kinetic_opal_block": {
+    "miningmaster:divine_beryl_block": {
       "trigger": "minecraft:inventory_changed",
       "conditions": {
         "items": [
           {
             "items": [
-              "miningmaster:kinetic_opal_block"
+              "miningmaster:divine_beryl_block"
+            ]
+          }
+        ]
+      }
+    },
```

Comment:
If I'm reading this correctly, you removed the Kinetic Opal Block from this advancement, why? I don't see it added anywhere else in this file so it seems to be gone altogether.

Reply:
Oh whoops! It looks like in copy-pasting for the other gems, I accidentally glazed over and replaced the last remaining KO entry on this one. My bad!